benchmarks/synchrobench/src/main/java/com/yahoo/oak/synchrobench/maps/BenchOakMap.java

```diff
@@ -119,23 +118,23 @@ public float nonHeapAllocatedGB() {
 
     /**
      * Iterate over a map and consume the keys/values.
-     * @param subMap subset of the map
+     * @param map map, or its subset
```

Comment:
One "map" is a typo?

Reply:
no, it's the name of the function parameter